### DISCUSSION TITLE
The Google Colab Notebook stuck forever at os.system("wget -qnc https://files.ipd.uw.edu/krypton/ananas")

### DISCUSSION THREAD

**Discussion by Thomas8081** [RosettaCommons/RFdiffusion#228] [Q&A] [answered]
Hi, I'm using the Google Colab Notebook of RFdiffusion. 
The program stuck at this line forever:  os.system("wget -qnc https://files.ipd.uw.edu/krypton/ananas").
Could anyone please tell me how to fix it and what's wrong?

I'm using the Google Colab Notebook of RFdiffusion (the link can be found RFdiffusion's github frontpage. I also paste it here: 
https://colab.research.google.com/github/sokrypton/ColabDesign/blob/v1.1.1/rf/examples/diffusion.ipynb).
It was great!
However, begining 2 or 3 days ago, the program stuck forever at this step (in the section of 'setup RFdiffusion (~3min)'):
os.system("wget -qnc https://files.ipd.uw.edu/krypton/ananas").
Could anyone please tell me how to fix it and what's wrong?

Thanks!

-------
I just tried to 'ping files.ipd.uw.edu' and got a 'timeout' message.
'ping ipd.uw.edu' was fine.
So it seemed that the 'files' server of ipd had some issue(s).

**Comment by roccomoretti** [comment, marked as answer]
Yes, the UW servers were undergoing maintenance over the weekend. They should be back up sometime soon.
